From: Ben Hutchings <ben@decadent.org.uk>
Date: Sat, 5 Jun 2010 00:42:57 +0100
Subject: [PATCH] Revert "ipv4: Make INET_LRO a bool instead of tristate."

This reverts commit e81963b180ac502fda0326edf059b1e29cdef1a2.

This code is now deprecated in favour of GRO, and only a few drivers
use it.  For Debian kernels (and probably other distribution kernels)
it should be built as a module.

The original change to prevent building it as a module was made in an
attempt to avoid the case where some dependents are set to y and some
to m, and INET_LRO can be set to m rather than y.  However, the
Kconfig system will now reliably set INET_LRO=y in this case.
---
 net/ipv4/Kconfig |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/net/ipv4/Kconfig b/net/ipv4/Kconfig
index 70491d9..483d427 100644
--- a/net/ipv4/Kconfig
+++ b/net/ipv4/Kconfig
@@ -398,7 +398,7 @@ config INET_XFRM_MODE_BEET
 	  If unsure, say Y.
 
 config INET_LRO
-	bool "Large Receive Offload (ipv4/tcp)"
+	tristate "Large Receive Offload (ipv4/tcp)"
 	default y
 	---help---
 	  Support for Large Receive Offload (ipv4/tcp).
-- 
1.7.1

